lib/wallet/integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -2248,7 +2193,8 @@ selectCoinsWith ctx w payments transform = do
         (Link.selectCoins @style w) Default (transform payload)
 
 delegationFee
-    :: forall w m. (HasType (ApiT WalletId) w, MonadIO m, MonadUnliftIO m)
+    :: forall w m
+     . (HasType (ApiT WalletId) w, MonadUnliftIO m)
```

Comment:
```suggestion
    :: forall w m. (HasType (ApiT WalletId) w, MonadUnliftIO m)
```

Reply:
I actually did it intentionally :) +1 more reason to have an auto-formatter :-)